src/Microsoft.Security.Utilities.Core/IdentifiableScan.cs

```diff
@@ -123,7 +125,19 @@ public void Start()
     {
         if (this.scan == IntPtr.Zero)
         {
-            this.scan = identifiable_scan_create();
+            var builder = new StringBuilder();
+
+            foreach (var id in this.idToLengthMap.Keys)
```

Comment:
>var [](http://example.com/codeflow?start=21&length=3)

we'd typically be explicit about type here,

Reply:
Moved to ThreadStatic as suggested offline.